backend/tests/regression/answer_quality/search_test_config.yaml

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Another good thing to put in the README is to update this file. The README should contain all of the info needed to run the tests without exploring the code.